### DISCUSSION TITLE
Turning Code Frequency Scale to logarithmic?

### DISCUSSION THREAD

**Discussion by StefanSchubert** [community/community#34878] [Repositories]
Hi,

when having major refactorings one may move a lot of code resulting in peaks on the frequency graph which will make to complete graph information unreadable having a linear scale:

<img width="1419" alt="image" src="https://user-images.githubusercontent.com/10548043/193466238-80d95fd7-77de-4196-85f4-7b778ee482f8.png">

Would it help if the graph uses a logarithmic scale instead?

Cheers
Stefan

**Comment by AdityaMayukhSom** [comment]
Either adding logarithmic scale, or the ability to only focus on a smaller time frame, both could help avoid this issue.

In case time boundary is added, then the graph could be drawn proportionately each time the time frame is changed.
